src/core/hle/service/apt/apt.h

```diff
@@ -42,6 +43,8 @@ struct CaptureBufferInfo {
 };
 static_assert(sizeof(CaptureBufferInfo) == 0x20, "CaptureBufferInfo struct has incorrect size");
 
+static const std::size_t SysMenuArgSize = 0x40;
```

Comment:
```suggestion
constexpr std::size_t SysMenuArgSize = 0x40;
```

static can be omitted, since file-scope const/constexpr has internal linkage by default.